src/cmds/envs.js

```diff
@@ -62,11 +63,40 @@ function list() {
     return console.log('There are no stored environments.')
   }
 
-  const rows = [
-    ['name', ...ENV_CONFIG_FIELDS],
-    ...envsKeys.map(envName => [envName, ...ENV_CONFIG_FIELDS.map(key => envs[envName][key] ?? '(Not Set)')])
-  ]
-  console.log(cliff.stringifyRows(rows, ['blue']))
+  const table = new Table({
+    style: {
```

Comment:
Also I had to copy `style` for each instance to have *borderless* style of table that we had before.
I thinking about either extract it to some config (not sure where is the best place to store it), or remove it completely and use single line default borders instead.